go.mod

```diff
@@ -58,6 +58,7 @@ require (
 	github.com/soheilhy/cmux v0.1.4 // indirect
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.3
+	github.com/stretchr/testify v1.3.0
```

Comment:
I am a bit hesitant to introduce `testify` as a dependency here. Looking at the changes in `web/srv/api_handlers_test.go`, I don't think we are gaining much, since `require.NoError()` is just [`if err != nil`](https://github.com/stretchr/testify/blob/a88bf7aab8f390daf0a71952568272f6638b319e/assert/assertions.go#L1225-L1227) and `assert.Equal()` is just `==`.

@scottcarol WDYT?

Reply:
I've only used some basic helpers on that test, but the `assert` and `require` packages contain [a lot](https://godoc.org/github.com/stretchr/testify/assert) of helpful helpers that may save a lot of boilerplate when writing tests. Even in this small example it's saving a few `if` blocks, making tests very easy to read. It'd be great if you could consider introducing it to the project :)

Reply:
@tegioz Could you open an issue proposing using this dependency? That way other Go developers on the project could chime in with feedback. For this specific PR, let's omit `testify` as a dependency, and then if we do have consensus on adding that dependency to our codebase, we can systematically update the tests. 😃

Reply:
Sure, no problem 👍